qdrant-landing/content/documentation/fastembed/fastembed-colbert.md

```diff
@@ -5,33 +5,58 @@ weight: 6
 
 # How to Generate ColBERT Multivectors with FastEmbed
 
-With FastEmbed, you can use ColBERT to generate multivector embeddings. ColBERT is a powerful retrieval model that combines the strength of BERT embeddings with efficient late interaction techniques. FastEmbed will provide you with an optimized pipeline to utilize these embeddings in your search tasks.
+Qdrant supports [multivector representations](https://qdrant.tech/documentation/concepts/vectors/#multivectors). 
+With FastEmbed, you can use ColBERT to generate multivector embeddings; FastEmbed will provide an optimized pipeline to utilize these embeddings in your search tasks.
 
-Please note that ColBERT requires more resources than other no-interaction models. We recommend you use ColBERT as a re-ranker instead of a first-stage retriever. 
+ColBERT produces one vector per token (a `token` is a meaningful text unit for a machine learning model). Consequently,
+ColBERT is more precise than dense embedding models like `BAAI/bge-small-en-v1.5`, which embed a whole text into just a single vector. 
+For the same reason, ColBERT requires significantly more resources than dense embedding models, so it should be primarily used for reranking rather than first-stage retrieval.
+A simple dense retriever can retrieve around 100-500 examples at the first stage; then, you can rerank them using ColBERT, moving the most relevant results to the top.
 
-The first-stage retriever can retrieve 100-500 examples. This task would be done by a simpler model. Then, you can rank the leftover results with ColBERT.
+ColBERT is a more production-suitable choice of a reranking model than [cross-encoders](https://sbert.net/examples/applications/cross-encoder/README.html).
```

Comment:
It's a claim, which might not be true, it depends on the use case, e.g. ColBERT consumer more memory and there are cases when memory is a bottleneck